.github/workflows/publish.js.yml

```
@@ -11,34 +11,25 @@ jobs:
   build:
     runs-on: macos-latest
     steps:
-    - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3
-    - name: Use Node.js 18.x
-      uses: actions/setup-node@e33196f7422957bea03ed53f6fbb155025ffc7b8 # v3
+    - uses: actions/checkout@v3
+    - name: Use Node.js LTS
+      uses: actions/setup-node@v3
       with:
-        node-version: 18.x
-    - run: npm ci
+        node-version: lts/*
+        check-latest: true
+    - run: npm install
       name: Install dependencies
-    - run: npm run build
-      name: Run build
     - run: npm run test
-      name: Run test
+      name: Run NPM Test
     - run: |
-        # Remove dev and peer dependencies from node_modules
-        npm prune --omit=dev --omit=peer
-
-        # Generate shrinkwrap from the content of node_modules
-        mv package-lock.json package-lock.json.bak
-        npm shrinkwrap
-        mv package-lock.json.bak package-lock.json
-
-        # Restore dev dependencies for the release
-        mv npm-shrinkwrap.json npm-shrinkwrap.json.bak
-        npm ci --only=dev
-        mv npm-shrinkwrap.json.bak npm-shrinkwrap.json
-
-        # Remove unnecessary package-lock.json
-        rm -f package-lock.json
-      name: Generate shrinkwrap without including dev/peer dependencies
+        rm -rf package-lock.json
+        npm prune --production
```

Here can keep omit as newer syntax

```suggestion
        npm prune --omit=dev --omit=peer
```

e.g. https://github.com/appium/appium-uiautomator2-driver/blob/master/.github/workflows/publish.js.yml

---

fixed